mitosheet/src/mito/components/endo/IndexHeaderContextMenu.tsx

```diff
@@ -20,11 +20,14 @@ export default function IndexHeaderContextMenu(props: {
     actions: Actions;
     closeOpenEditingPopups: (taskpanesToKeepIfOpen?: TaskpaneType[]) => void;
 }): JSX.Element {
+    const [ currentSubmenu, setCurrentSubmenu ] = React.useState<string | undefined>(undefined);
+
```

Comment:
I think we additionally want to set the current submenu to undefined when the parent dropdown closes. 

Otherwise, if you:
1. Open a dropdown
2. Mouse over to open a submenu
3. Open another dropdown
4. Open the original dropdown
5. The original submenu is open